apps/portal/src/content/docs/components/calendar.mdx

```diff
@@ -0,0 +1,180 @@
+---
+title: Calendar
+description: Calendar component documentation
+---
+
+The `Calendar` component provides a flexible UI element for displaying a date picker. It includes several customizations for styling and functionality.
+
+import { DocsPage } from "@/components/docs-page";
+
+<DocsPage.ComponentExample
+  client:only
+  code={`() => {
+  const [selectedDate, setSelectedDate] = React.useState<Date | undefined>([new Date()]);
+
+return (
+
+<Calendar
+  mode="range"
+  selected={selectedDate}
+  onSelect={setSelectedDate}
+  showOutsideDays={true}
+  fromDate={new Date(2023, 10, 12)}
+  toDate={new Date(2023, 11, 15)}
+  weekStartsOn={1}
+/>
+); }; `} />
+
+## Usage
+
+```typescript jsx
+import { useState } from 'react';
+import { Calendar } from '@harnessio/ui/components';
+
+const CalendarExample = () => {
+  const [selectedDate, setSelectedDate] = useState<Date | undefined>(new Date());
+
+  return (
+    <Calendar
+      mode="single"
+      selected={selectedDate}
+      onSelect={setSelectedDate}
+      showOutsideDays={true}
+      className="custom-calendar"
+      classNames={{
+        day: 'custom-day',
+        selected: 'custom-selected',
+        today: 'custom-today',
+      }}
+      components={{
+        Day: ({ date, selected, ...props }) => (
+          <div {...props} className={selected ? 'custom-selected' : 'custom-day'}>
+            {date.getDate()}
+          </div>
+        ),
+      }}
+      disabled={{ from: new Date(2023, 0, 1), to: new Date(2023, 0, 10) }}
+      fromDate={new Date(2023, 0, 1)}
+      toDate={new Date(2023, 11, 31)}
+      locale={navigator.language}
+      weekStartsOn={1}
+    />
+  );
+};
+
+export default CalendarExample;
+```
+
+## Anatomy
+
+All parts of the Calendar component can be imported and composed as required.
+
+```typescript jsx
+<Calendar
+  mode="single"
+  selected={new Date()}
+  onSelect={(date) => alert(\`Selected date: \${date}\`)}
+  showOutsideDays={true}
+  className="custom-calendar"
+  classNames={{
+    day: 'custom-day',
+    selected: 'custom-selected',
+    today: 'custom-today',
+  }}
+  components={{
+    Day: ({ date, selected, ...props }) => (
+      <div {...props} className={selected ? 'custom-selected' : 'custom-day'}>
+        {date.getDate()}
+      </div>
+    ),
+  }}
+  disabled={{ from: new Date(2023, 0, 1), to: new Date(2023, 0, 10) }}
+  fromDate={new Date(2023, 0, 1)}
+  toDate={new Date(2023, 11, 31)}
+  locale={navigator.language}
+  weekStartsOn={1}
+/>
+```
+
+## API Reference
+
+### Calendar
+
+The `Calendar` component serves as the main container for the date picker.
```

Comment:
We can drop this too since it's already covered in the introduction and is also kinda wrong since it's not a _container_ for the calendar, it _is_ the calendar.
```suggestion
```